frontend/src/services/interceptors.ts

```diff
@@ -21,7 +21,7 @@ export function comsAxios(timeout = 10000) {
   instance.interceptors.request.use(
     (cfg: any) => {
       if (authStore.ready && authStore.getAuthenticated) {
```

Comment:
Should getAuthenticated be authStore.getKeycloak.authenticated instead?

Reply:
Yeah good catch (this would work anyways because the authenticated stays set until you log out, whereas refresh token changes, but should update to be consistent)